mmv1/templates/terraform/examples/backend_service_cache_include_named_cookies.tf.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_compute_backend_service" "<%= ctx[:primary_resource_id] %>" {
+  name          = "<%= ctx[:vars]['backend_service_name'] %>"
+  enable_cdn  = true
+  cdn_policy {
+    cache_mode = "CACHE_ALL_STATIC"
+    default_ttl = 3600
+    client_ttl  = 7200
+    max_ttl     = 10800
+    include_named_cookies = ["_bypass_cache"]
```

Comment:
i believe this should be nested under `cache_key_policy`. is that correct?

Reply:
yes